ts/torch_handler/utils/optimization/ipex_optimization.py

```diff
@@ -0,0 +1,88 @@
+from .optimization import optimization_registry, Optimization
+from ..conf.ipex_config import Conf
+import os
+import torch
+import logging
+import subprocess 
+
+logger = logging.getLogger(__name__)
+
+@optimization_registry
+class IPEXOptimization(Optimization):
+    """The Intel® Extension for PyTorch* (IPEX) Optimization.
+    
+    Args:
+        cfg (Conf): the optimization configuration.
+    """
+    def __init__(self, cfg):
+        super().__init__(cfg)
+        
+        self.dtype = cfg.dtype
+        self.channels_last = cfg.channels_last
+        
+        if self.dtype == 'int8':
+            self.quantization_approach = cfg.quantization.approach
+            self.quantization_example_inputs = cfg.quantization.example_inputs
+            self.quantization_calibration_dataset = cfg.quantization.calibration_dataset
+        
+        self.torchscript = False 
+        if cfg.torchscript.approach is not None:
+            self.torchscript = True 
+            self.torchscript_approach = cfg.torchscript.approach
+            self.torchscript_example_inputs = cfg.torchscript.example_inputs
+
+    def optimize(self, model):
+        """Apply Intel® Extension for PyTorch* (IPEX) optimizations to the given model (nn.Module).
+           
+        Args:
+            model (torch.nn.Module): The model to optimize. 
+        Returns:
+            torch.nn.Module: The optimized model.
+        """
+        import intel_extension_for_pytorch as ipex
+        
+        # channel last 
+        if self.channels_last:
+            model = model.to(memory_format=torch.channels_last)
+        
+        # dtype 
+        if self.dtype == 'float32':
+            model = ipex.optimize(model, dtype=torch.float32)
+        elif self.dtype == 'bfloat16':
+            model = ipex.optimize(model, dtype=torch.bfloat16)
+        else: # int8
+            from intel_extension_for_pytorch.quantization import prepare, convert
+            
+            if self.quantization_approach == 'static':

```

Comment:
Thanks @min-jean-cho for the updates, I believe including quantization or tracing on the fly as part of serving may raise some concerns. For example quantization may result in accuracy drop that would be great if user can provide the quantized model to make sure we are not causing the drop on the fly without having the chance to check. 

Similarly tracing the model may fail as well for different sort of reasons. 

It might be safer if we leave the optimizations that changes the models weight the (the .pt)/ kernel fusions as an example as its been [here](https://github.com/pytorch/serve/tree/master/examples/intel_extension_for_pytorch#creating-and-exporting-int8-model-for-ipex) or in a separate optimization folder with automated scripts offerings that we could extend in future with other optimizations as well.

Then we can keep anything that does not apply changes to model weights/ kernels (fusions) such as core pinning and similar optimizations as part of the base_handler.

This would also let us keep the config files focused on runtime settings and not mixing it with non-runtime optimizations.